src/audio/sr_audio/sr_audio.c

```diff
@@ -0,0 +1,720 @@
+// Copyright(c) 2021 Sound Research Corporation. All rights reserved.
```

Comment:
2022 ?

Reply:
Well, the code was written back in 2021 but we can change it to 2022 if needed